python_client/test.py

```diff
@@ -0,0 +1,56 @@
+
+# make install
+
+# currently CMAKE_INSTALL_LIBDIR=/home/ENV{USERNAME}/chronolog/lib
+
+## create symlink for  py_chronolog_client.[linux_distibution_version].so 
+## ln -s ${CMAKE_INSTALL_LIBDIR}/py_chronolog_client.so ${CMAKE_INSTALL_LIBDIR}/py_chronolog_client.[linux_distribution_version].so
```

Comment:
the last argument should be the link name. the second to the last argument should be the target name.